plugins/agent-utilities/skills/fusion-parametric-design/tests/test_mesh_segmentation.py

```
@@ -1116,16 +1161,137 @@ def test_a_region_below_the_power_floor_says_so_and_does_not_claim_the_gate(self
+        """
+        import struct
+
+        vertices, triangles, groups = quantized_bore_mesh()
+        floor = seg.fit_regions(make_dump(vertices, triangles, groups), spec(min_feature_size=1.0))[
+            "noise"
+        ]["vertex_precision_floor"]
+
+        bore_group = max(groups)
+        first_bore_vertex = min(
+            index
+            for triangle, group in zip(triangles, groups)
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)